### ISSUE TITLE
About the data_preprocess

### ISSUE THREAD

**Issue by Magicyu-2015** [Deep-imagelab/AWAN#1]
Dear Author, Thank you for providing a high-quality code to reproduce your research. However, I have some questions about data preprocessing.
In your Python code 'train_data_preprocess.py':
```python
mat = h5py.File(filenames_hyper[k], 'r')
hyper = np.float32(np.array(mat['rad']))
```
In the original MATLAB mat file. no keyword 'rad' is provided. Instead, it provides the 'cube', 'bands' and 'norm_factor'.  Do you preprocess the original MATLAB file and add the key name 'rad'?

**Comment by Deep-imagelab** [comment]
I'm so sorry for making such a miskate.  It should be the key name  'cube' rather than 'rad'. 
Thank you.

**Comment by jovian-2wang** [comment]
Hello, can you share your training data set? I can't find this part of the data set from the Internet. Thank you

**Comment by yazdaniamir38** [comment]
Hello, I'm trying to train your model from scratch, however after about 50 epochs the train loss (MRAE loss) gets negative, and the training doesn't converge. Do you have any idea why that happens? I'm using exactly the same parameters as yours. thank you!
